tests/sentry/api/test_event_search.py

```diff
@@ -192,6 +192,7 @@ def run_test_case(self, name, case):
     "timestamp_rollup",
     "has_tag",
     "not_has_tag",
+    "supported-tags",
```

Comment:
Should this be an underscore, all the others are

Reply:
Done, and fixed the other one that wasn't underscored